app/services/subscriptions/free_trial_billing_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Subscriptions
+  class FreeTrialBillingService < BaseService
+    def initialize(timestamp: Time.current)
+      @timestamp = timestamp
+
+      super
+    end
+
+    def call
+      ending_trial_subscriptions.each do |subscription|
+        # if subscription.plan_pay_in_advance
```

Comment:
Layout/CommentIndentation: Incorrect indentation detected (column 8 instead of 10).